### DISCUSSION TITLE
Help Needed: Account Flagging and Access Issues Despite Security Updates

### DISCUSSION THREAD

**Discussion by HonsalSub** [community/community#129364] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello GitHub Community,

I'm reaching out due to persistent problems with my account that began after it was compromised. Despite updating all security measures and contacting support, the issues remain unresolved, affecting both my ability to use services and billing.

However that account that issue occurred is unable to write community post, so I am writing in backup account.

### Background:
- **Account Compromise:** My account was initially hacked, leading to unauthorized activities that were later detected, prompting an official security alert from GitHub.
- **Security Measures:** Following the alert, I promptly changed my password and updated all related security settings, including passkeys, 2FA, and SMS verification.
- **Access and Flagging:** After securing my account, I tried to access GitHub Copilot, which I had been subscribed to. However, I found the service inaccessible, my account was flagged, and I couldn't make any direct purchases from the plan comparison page due to this flag.

### Current Status:
- **Persistent Flagging:** Despite resolving the security issues, my account remains flagged, which blocks me from making purchases and accessing my subscriptions.
- **Billing Anomalies:** Ironically, while I cannot purchase anything or use my GitHub Copilot due to the flagged state, my subscription was still renewed and charged today.
- **Lack of Support Response:** I have not received any updates or responses to my follow-up communications regarding support ticket (#2838675).

### Specific Issues:
- **Billing:** My account continues to be charged for services like the subscription renewal, despite being flagged and thus unable to access these services.
- **Access:** I am completely locked out of using essential tools like GitHub Copilot, which my work heavily relies on.

### Questions for the Community:
1. Has anyone else experienced prolonged flagging after resolving security issues from a hack? How did you resolve it?
2. What can be done to escalate the issue within GitHub when standard support responses fail?
3. Are there known issues or tips for dealing with account recovery and billing inconsistencies after security breaches?

---

Your insights or any shared experiences regarding similar issues would be immensely helpful as I navigate these challenges. Thank you for any guidance you can provide!

**Comment by Honsal** [comment]
THIS IS MY MAIN ACCOUNT. For some reason, it seems to be able to write comment with flagged account.

However, GitHub supports is not responding until now. Where did my money go? It is ridiculous that this account is flagged but monthly subscription is still paying, and I cannot refund because lack of support response.
